Bambu/source/FatJetsFiller.cc

```diff
@@ -21,9 +21,28 @@ mithep::nero::FatJetsFiller::defineBranches(TTree* _tree)
 }
 
 void
+mithep::nero::FatJetsFiller::initialize()
+{
+	topANN = new NeuralNet(5,2);
+	#include "topTagger_simple.icc" // these are just weights - move to MIT_DATA?
```

Comment:
I agree, it's better to eventually make it into a plain text table or xml or whatever.